src/renderer/app/App.tsx

```diff
@@ -22,6 +24,10 @@ const App = () => {
   const appRoutes = useRoutes(routesConfig);
   const { getAccounts } = useAccount();
 
+  const [assetsPrices, activeCurrency] = useUnit([priceProviderModel.$assetsPrices, currencyModel.$activeCurrency]);
```

Comment:
This is added temporary, to make effector model work you need to import it somewhere.
Can be deleted after Egor's PR